proposals/2019/Add DaylightingIlluminanceSetpointUnits element.md

```diff
@@ -0,0 +1,23 @@
+# Add DaylightingIlluminanceSetpointUnits Element
+
+## Overview
+
+The `auc:DaylightingIlluminanceSetpoint` element should have a corresponding "units" element.
+
+## Justification
+
+Currently, the units of measure for the `auc:DaylightingIlluminanceSetpoint` element is given in lux.
+Other units of measure are defined in the literature, e.g., foot-candle [1].
+
+## Implementation
+
+This proposal is to add a new `auc:DaylightingIlluminanceSetpointUnits` element as an enumeration with the following values:
+* "fc" (Foot-candle)
+* "lux" (Lumens per square meter)
```

Comment:
In general BSync does not use SI units at all, so I think we want just be explicit in the annotation that the units are foot-candles. Thoughts @markborkum?

Reply:
@nllong This is definitely more straightforward than adding a new enumeration. Not very inclusive for users outside of USA, Liberia and Myanmar, on the other hand.

Reply:
agreed that this is not friendly to most of the world. But it is what it is. We can create BSync-SI.xsd someday.